initializr-service/src/main/resources/application.yml

```diff
@@ -951,6 +958,67 @@ initializr:
+          description: Spring Security integration with Azure Active Directory for authentication
+          links:
+            - rel: guide
+              href: https://github.com/Microsoft/azure-spring-boot/tree/master/azure-samples/azure-ad-integration-spring-boot-starter-sample
+              description: Sample code for using the starter
+            - rel: reference
+              href: https://github.com/Microsoft/azure-spring-boot/tree/master/azure-starters/azure-ad-integration-spring-boot-starter
+              description: Source code and README for the starter
+        - name: Azure Cosmos DB (DocumentDB)
+          id: document-db
+          groupId: com.microsoft.azure
+          artifactId: azure-documentdb-spring-boot-starter
+          description: Spring Boot starter for Azure Document DB service
+          links:
+            - rel: guide
+              href: https://github.com/Microsoft/azure-spring-boot/tree/master/azure-samples/azure-documentdb-spring-boot-starter-sample
+              description: Sample code for using the starter
+            - rel: reference
+              href: https://github.com/Microsoft/azure-spring-boot/tree/master/azure-starters/azure-documentdb-spring-boot-starter
+              description: Source code and README for the
+        - name: Azure Key Vault
+          id: key-vault
+          groupId: com.microsoft.azure
+          artifactId: azure-keyvault-secrets-spring-boot-starter
+          description: Spring value annotation integration with Azure Key Vault Secrets
+          links:
+            - rel: guide
+              href: https://github.com/Microsoft/azure-spring-boot/tree/master/azure-samples/azure-keyvault-secrets-spring-boot-starter-sample
+              description: Sample code for using the starter
+            - rel: reference
+              href: https://github.com/Microsoft/azure-spring-boot/tree/master/azure-starters/azure-keyvault-secrets-spring-boot-starter
+              description: Source code and README for the starter
+        - name: Azure Storage
+          id: azure-storage
+          groupId: com.microsoft.azure
+          artifactId: azure-storage-spring-boot-starter
+          description: Spring Boot starter for Azure Storage service
+          links:
+            - rel: guide
+              href: https://github.com/Microsoft/azure-spring-boot/tree/master/azure-samples/azure-storage-spring-boot-starter-sample
+              description: Sample code for using the starter
+            - rel: reference
+              href: https://github.com/Microsoft/azure-spring-boot/tree/master/azure-starters/azure-storage-spring-boot-starter
+              description: Source code and README for the starter
+        - name: Azure Support
+          id: azure-support
+          groupId: com.microsoft.azure
+          artifactId: azure-support
+          description: Auto-configuration for Azure services like Service Bus and Cloud Foundry support
```

Comment:
Auto-configuration for Azure Services (service bus, storage, active directory, cosmos DB, key vault and more).